mtoken-model/src/main/java/com/wultra/security/powerauth/lib/mtoken/model/entity/Operation.java

```diff
@@ -38,6 +38,7 @@ public class Operation {
     private String name;
     @NotNull
     private String data;
+    @NotNull
```

Comment:
🤔 Should it be `@NotNull` only for `/operation/history` or also for `/operation/list`, `/operation/detail`, and `/operation/detail/claim`?